.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,44 @@
+name: Lint changes in PR
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+
+jobs:
+  lint-nodejs-pr:
+    name: Lint NodeJS PR - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Lint checks don't differ according to the OS. So, we don't need different OS here.